docs/api/app.md

```diff
@@ -408,6 +408,8 @@ non-minimized.
 This event is guaranteed to be emitted after the `ready` event of `app`
 gets emitted.
 
+**Note:** It is only emitted when the second instance calls `app.requestSingleInstanceLock()`.
```

Comment:
Suggestion: this seems to be relevant enough information to be part of the main paragraph. Currently, the first sentence is incomplete.

Reply:
please check again